src/main/scala/com/databricks/labs/overwatch/pipeline/PipelineTargets.scala

```diff
@@ -518,6 +518,22 @@ abstract class PipelineTargets(config: Config) {
       config
     )
 
+    lazy private[overwatch] val notebookCommandsTarget: PipelineTable = PipelineTable(
+      name = "notebookCommands_gold",
+      _keys = Array("notebook_id", "unixTimeMS"),
```

Comment:
shouldn't the command_id also be a key of this table?

Reply:
umm but command_id would be having unique value for each row and if we make it as key wouldn't it make the indexing slower?

Reply:
I was thinking more of what if we have multiple very small commands that come in at the exact same unixTimeMS, we would end up with only the first one we see, no? But walking through the scenario I think it's not very likely, since this is at the millisecond level.